templates/identity/email-password/form-script-partial.html.ejs

```diff
@@ -0,0 +1,41 @@
+<script>
+  const form = document.getElementById('mainForm');
+  form.addEventListener('submit', postForm);
+
+  const errorBlock = document.getElementById('error');
+
+  async function postForm(event) {
+    event.preventDefault();
+
+    const formData = new FormData(form);
+    const res = await fetch('', {
+      method: 'POST',
+      credentials: 'include',
+      headers: { 'accept': 'application/json', 'content-type': 'application/json' },
+      body: JSON.stringify(Object.fromEntries(formData)),
+    });
+    if (res.status >= 400) {
+      const error = await res.json();
+      errorBlock.innerText = `${error.statusCode} - ${error.name}: ${error.message}`;
+    } else if (res.status === 200 || res.status === 201) {
+      const body = await res.json();
+      if (body.location) {
+        window.location.href = body.location;
+      } else {
+        console.log(body);
```

Comment:
oops

Reply:
Yea we really could also use some linting in these files 😅